### DISCUSSION TITLE
Regrading single coil knee test dataset for public leaderboard submission

### DISCUSSION THREAD

**Discussion by SaiPavan-Tadem** [facebookresearch/fastMRI#292] [Q&A]
Three months ago, I downloaded the training and validation single coil data for my experiments. I'm now ready to submit my results to the leaderboard, but I noticed that there have been some changes to the fastMRI dataset recently. Previously, the size of the test dataset was around 7 GB, but now it's about 1.4 GB. I'm wondering if this is just a compressed version of the previous test data, and whether I can use it to evaluate my model, which was trained on the dataset I downloaded three months ago. Can you please clarify if this is possible?

**Comment by mmuckley** [comment]
Hello @SaiPavan-Tadem, are you talking about the size of the `.tar.xz` file or the size of the unzipped data?

**Reply by SaiPavan-Tadem** [reply]
@mmuckley , Yes .tar.xz files, In the Public Leaderboard Submission (knee single coil) section,To summarise, I have two queries:
 1. It is specified that we must use the v2_test data. Can you please provide information on where to find this data? 
2. Has the Single Coil Dataset changed? I noticed that the number of files in the dataset remains the same, but the file names are different.

**Reply by mmuckley** [reply]
1. Yes, v2 test data is the same as v1 essentially. It's just that in v1 the zeros were signed, which gave the networks some learning signal. You just need to change the filenames for leaderboard upload.
2. It shouldn't have changed. Did you also verify that the data changed? If so then I will look into converting it back to the correct dataset.

**Comment by SaiPavan-Tadem** [comment]
1. What is v1 test data ? Could you confirm if I am using the correct data
for train, val, and test (for submission)? I have attached an email from
NYU with links for data downloads. Please let me know if I need to download
any additional files
2. I need to check the patient information .

**Reply by mmuckley** [reply]
Hello @SaiPavan-Tadem 

1. The "v1" test data is something from the earliest days of fastMRI. In the original release, the masked data was constructed as `masked_kspace = mask * full_kspace`. The issue is by default the 0s are signed (either -0.0 or +0.0), and the models could learn a little bit from this leaked information. The "v2" test data fixed this by using the `torch.where` operation instead. So you (and probably everyone) likely has the v2 test data, and you can tell by looking at the masked reasons and seeing if the zeros are signed or unsigned.
2. If the filenames match then it should be okay. Otherwise there may have been a bug in this conversion.
